src/i18n/loadStepContent.js

```diff
@@ -76,6 +76,7 @@ const loadStepContent = (step, i18Next) => {
   };
 
   const promises = [
+    glob('common/content.json').then(loadContents),
```

Comment:
Might be worth doing this in i18Next.js i18Middleware, so that it runs only once on startup instead of on every request.

I'm not sure exactly but I think I set it up so the `translations` namespace would be the fallback, so you could use `loadFileContents` instead of `loadContents`. i18Next definitely allows for that.

Also (personal preference and I'm not your user of course) but being able to pass to journey a file path or glob would be really nice:

```
journey(app, {
  content: 'common/*.json',
  ...
});
```
that way content designers can organise their common content to their hearts content.

Reply:
not sure exactly how i'd do that, but `ive added *.json anyway